zellij-client/src/cli_client.rs

```diff
@@ -28,6 +28,10 @@ pub fn start_cli_client(os_input: Box<dyn ClientOsApi>, session_name: &str, acti
                 os_input.send_to_server(ClientToServerMsg::ClientExited);
                 process::exit(0);
             },
+            Some((ServerToClientMsg::TabNameList(tab_names), _)) => {
```

Comment:
Hey @naosense - let's change `ServerToClientMsg::TabNameList` to `ServerToClientMsg::Log`. In `cli_client` it will do exactly what it does here, and in the normal client it will log to file (with `log::info`).

Reply:
ok, one more question, what does a normal client refer to?

Reply:
Yeah, let's make things consistent.

The "normal" client is the UI, eg. the client that sends your keyboard clicks and actions to the server. This is as opposed to the cli client which does this from the command line. I want us to handle this instruction in both places properly, so that users will be able to eg. use `QueryTabList` as a keyboard binding too if they want, and then the tab list will appear in the log file.

You can find it here: https://github.com/zellij-org/zellij/blob/f8a078c1a959ca7bb243bd9a86b3ddc392605c9c/zellij-client/src/lib.rs#L391

Makes sense?

Reply:
yeah, thanks for your reply